tests/argocd_test.go

```diff
@@ -0,0 +1,65 @@
+package bundles_test
+
+import (
+	"os"
+	"path/filepath"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("argocd test", Label("argocd"), func() {
+
+	BeforeEach(func() {
+		prepareBundle()
+	})
+	AfterEach(func() {
+		cleanBundle()
+	})
+
+	It("Deploy argocd with default version", func() {
+		runBundle()
+		dat, err := os.ReadFile(filepath.Join("/var/lib/rancher/k3s/server/manifests", "argocd.yaml"))
+		content := string(dat)
+		Expect(err).ToNot(HaveOccurred())
+		Expect(content).To(MatchRegexp("version: \".*?\""))
```

Comment:
should this match the actual version set as default? or is the only requirement here that it has a value?

Reply:
It should match the default version